xml/System.Diagnostics/Activity.xml

```diff
@@ -446,8 +446,8 @@ If the `DisplayName` property is not set, it will return the same value as `Oper
         <Parameter Name="disposing" Type="System.Boolean" Index="0" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="disposing">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="disposing">Usually, be false if the disposing occurring in the finalizer and true if called from Dispose method.</param>
+        <summary>This method is provided to allow the Activity subclassing overrides it and release any allocated resources done by this subclassing code.</summary>
```

Comment:
@gewarren are we allowed/supposed to mention that the Dispose method is virtual so the derived type should ensure the resources get released?:
```suggestion
        <summary>When overriden by a deriven type, this method releases any allocated resources.</summary>
```